### ISSUE TITLE
Support bridging

### ISSUE THREAD

**Issue by josephroberts** [resin-io/resin-wifi-connect#66]
Allow the wifi internet connection to be shared with an external device over ethernet

**Comment by bbinet** [comment]
Yes this would be great!
It would also be very useful to us if the wifi could be used both as an access point (resin-wifi-connect) and a wifi client at the same time when the wifi device supports this feature.
Should I create a separate issue to track this feature?

Here is a post which explains how to setup this wifi network configuration on the rpis:
http://imti.co/post/145442415333/raspberry-pi-3-wifi-station-ap

**Comment by majorz** [comment]
@bbinet right, it is indeed possible on some device running in both AP mode and as a regular WiFi client at the same time. You may create a separate issue for this, so that we keep track of it.

**Comment by nagualcode** [comment]
what about sharing the ethernet connectivity to clients connected to the AP ?

**Comment by bbinet** [comment]
@majorz Is there any manual way to force wificonnect to share internet connection to clients connected to the AP?
